independent-publisher-connectors/ClickUp Team Manager/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",

```

Comment:
@marsurice The default iconBrandColor for any IP connector is "#da3b01". Please update it.